### PR TITLE
135 some namespaces attributions has to be reviewed

### LINKED ISSUE #135: Some namespaces attributions has to be reviewed
Some components, systems, functions are badly put in namespaces

Examples: the functions to create predefined entities are in the namespace ecs but they aren't in the ECS Library.

### PR DESCRIPTION
**Pull request by ZQUEMA**
## Updated features
- Change the namespace of the Graphic-SFML library from `ecs` to `graphic::sfml`
- Change the namespace of the Trasisthor ECS Logic library from `ecs` to `transisthor::ecslogic`
- Change the namespace of the RType Logic libraries (Global, Server, EntityManipulation) from `ecs` or nothing to `rtypelogic::global/server/entitymanipulation`

close #135

### CODE REVIEW

File: libs/Transisthor/ECSLogic/Client/Systems/SendNewlyCreatedToServer.hpp [hunk 1]
```diff
@@ -0,0 +1,22 @@
+/*
+** EPITECH PROJECT, 2022
+** R-Type
+** File description:
+** SendNewlyCreatedToServer
+*/
+
+#pragma once
+
+#include "World/World.hpp"
```
Comment (31Nathan) [inline libs/Transisthor/ECSLogic/Client/Systems/SendNewlyCreatedToServer.hpp:10]: Missing jumpline

File: libs/Transisthor/ECSLogic/Server/Systems/SendToClient.hpp [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+#include "R-TypeLogic/Global/Components/DeathComponent.hpp"
+#include "R-TypeLogic/Global/Components/DestinationComponent.hpp"
+#include "R-TypeLogic/Global/Components/EquipmentComponent.hpp"
+#include "R-TypeLogic/Global/Components/InvinsibleComponent.hpp"
+#include "R-TypeLogic/Global/Components/InvisibleComponent.hpp"
+#include "R-TypeLogic/Global/Components/LifeComponent.hpp"
+#include "R-TypeLogic/Global/Components/PositionComponent.hpp"
+#include "R-TypeLogic/Global/Components/VelocityComponent.hpp"
+
+
```
Comment (31Nathan) [inline libs/Transisthor/ECSLogic/Server/Systems/SendToClient.hpp:26]: Useless empty line

**Review by 31Nathan — CHANGES_REQUESTED** [review]

File: libs/GraphicECS/SFML/Systems/InputManagement.cpp [hunk 3]
```diff
@@ -90,7 +92,7 @@ void InputManagement::run(World &world)
         }
     }
     for (auto &entityPtr : Inputs) {
-        std::queue<std::pair<ecs::ActionQueueComponent::inputAction_e, float>> &actions =
+        std::queue<std::pair<graphic::sfml::ActionQueueComponent::inputAction_e, float>> &actions =
```
Comment (osvegn) [inline libs/GraphicECS/SFML/Systems/InputManagement.cpp:95]: don't use "graphic::sfml" if you write "using namespace graphic::sfml" at the beginning of the file

File: libs/R-TypeLogic/EntityManipulation/CreateEntitiesFunctions/CreateAlliedProjectile.hpp [hunk 4]
```diff
@@ -24,7 +24,11 @@
 #include "R-TypeLogic/Global/Components/WeightComponent.hpp"
 #include "R-TypeLogic/Global/SharedResources/Random.hpp"
 
-namespace ecs
+using namespace transisthor::ecslogic;
+using namespace rtypelogic::global;
+using namespace ecs;
```
Comment (osvegn) [inline libs/R-TypeLogic/EntityManipulation/CreateEntitiesFunctions/CreateAlliedProjectile.hpp:29]: Useless

File: libs/R-TypeLogic/EntityManipulation/CreateEntitiesFunctions/CreateAlliedProjectile.hpp [hunk 5]
```diff
@@ -24,7 +24,11 @@
...
+
+namespace rtypelogic::entitymanipulation
```
Comment (osvegn) [inline libs/R-TypeLogic/EntityManipulation/CreateEntitiesFunctions/CreateAlliedProjectile.hpp:31]: update the comment at the end of the namespace

File: libs/R-TypeLogic/EntityManipulation/CreateEntitiesFunctions/CreateBackground.hpp [hunk 6]
```diff
@@ -17,7 +17,10 @@
 #include "R-TypeLogic/Global/Components/VelocityComponent.hpp"
 #include "R-TypeLogic/Global/Components/WeightComponent.hpp"
 
-namespace ecs
+using namespace rtypelogic::global;
+using namespace ecs;
+
+namespace rtypelogic::entitymanipulation
```
Comment (osvegn) [inline libs/R-TypeLogic/EntityManipulation/CreateEntitiesFunctions/CreateBackground.hpp:23]: update the comment at the end of the namespace

File: libs/R-TypeLogic/EntityManipulation/CreateEntitiesFunctions/CreateBackground.hpp [hunk 7]
```diff
@@ -17,7 +17,10 @@
 #include "R-TypeLogic/Global/Components/VelocityComponent.hpp"
 #include "R-TypeLogic/Global/Components/WeightComponent.hpp"
 
-namespace ecs
+using namespace rtypelogic::global;
+using namespace ecs;
```
Comment (osvegn) [inline libs/R-TypeLogic/EntityManipulation/CreateEntitiesFunctions/CreateBackground.hpp:21]: Useless

File: libs/R-TypeLogic/EntityManipulation/CreateEntitiesFunctions/CreateButton.hpp [hunk 8]
```diff
@@ -17,7 +17,10 @@
 #include "R-TypeLogic/Global/Components/VelocityComponent.hpp"
 #include "R-TypeLogic/Global/Components/WeightComponent.hpp"
 
-namespace ecs
+using namespace rtypelogic::global;
+using namespace ecs;
+
+namespace rtypelogic::entitymanipulation
```
Comment (osvegn) [inline libs/R-TypeLogic/EntityManipulation/CreateEntitiesFunctions/CreateButton.hpp:23]: update the comment at the end of the namespace

File: libs/R-TypeLogic/EntityManipulation/CreateEntitiesFunctions/CreateButton.hpp [hunk 9]
```diff
@@ -17,7 +17,10 @@
 #include "R-TypeLogic/Global/Components/VelocityComponent.hpp"
 #include "R-TypeLogic/Global/Components/WeightComponent.hpp"
 
-namespace ecs
+using namespace rtypelogic::global;
+using namespace ecs;
```
Comment (osvegn) [inline libs/R-TypeLogic/EntityManipulation/CreateEntitiesFunctions/CreateButton.hpp:21]: Useless

File: libs/R-TypeLogic/EntityManipulation/CreateEntitiesFunctions/CreateEnemy.hpp [hunk 10]
```diff
@@ -31,7 +31,11 @@
 #define MINIMUM_HEIGTH 0
 #define MAXIMUM_HEIGTH 600
 
-namespace ecs
+using namespace transisthor::ecslogic;
+using namespace rtypelogic::global;
+using namespace ecs;
```
Comment (osvegn) [inline libs/R-TypeLogic/EntityManipulation/CreateEntitiesFunctions/CreateEnemy.hpp:36]: Useless

File: libs/R-TypeLogic/EntityManipulation/CreateEntitiesFunctions/CreateEnemy.hpp [hunk 11]
```diff
@@ -31,7 +31,11 @@
...
+
+namespace rtypelogic::entitymanipulation
```
Comment (osvegn) [inline libs/R-TypeLogic/EntityManipulation/CreateEntitiesFunctions/CreateEnemy.hpp:38]: update the comment at the end of the namespace

File: libs/R-TypeLogic/EntityManipulation/CreateEntitiesFunctions/CreateEnemyProjectile.hpp [hunk 12]
```diff
@@ -24,7 +24,12 @@
 #include "R-TypeLogic/Global/Components/WeightComponent.hpp"
 #include "R-TypeLogic/Global/SharedResources/Random.hpp"
 
-namespace ecs
+using namespace transisthor::ecslogic;
+using namespace rtypelogic::global;
+using namespace transisthor::ecslogic;
+using namespace ecs;
```
Comment (osvegn) [inline libs/R-TypeLogic/EntityManipulation/CreateEntitiesFunctions/CreateEnemyProjectile.hpp:30]: Useless

File: libs/R-TypeLogic/EntityManipulation/CreateEntitiesFunctions/CreateEnemyProjectile.hpp [hunk 13]
```diff
@@ -24,7 +24,12 @@
...
+
+namespace rtypelogic::entitymanipulation
```
Comment (osvegn) [inline libs/R-TypeLogic/EntityManipulation/CreateEntitiesFunctions/CreateEnemyProjectile.hpp:32]: update the comment at the end of the namespace

File: libs/R-TypeLogic/EntityManipulation/CreateEntitiesFunctions/CreateObstacle.hpp [hunk 14]
```diff
@@ -24,7 +24,11 @@
 #include "R-TypeLogic/Global/Components/WeightComponent.hpp"
 #include "R-TypeLogic/Global/SharedResources/Random.hpp"
 
-namespace ecs
+using namespace rtypelogic::global;
+using namespace transisthor::ecslogic;
+using namespace ecs;
```
Comment (osvegn) [inline libs/R-TypeLogic/EntityManipulation/CreateEntitiesFunctions/CreateObstacle.hpp:29]: Useless

File: libs/R-TypeLogic/EntityManipulation/CreateEntitiesFunctions/CreateObstacle.hpp [hunk 15]
```diff
@@ -24,7 +24,11 @@
...
+
+namespace rtypelogic::entitymanipulation
```
Comment (osvegn) [inline libs/R-TypeLogic/EntityManipulation/CreateEntitiesFunctions/CreateObstacle.hpp:31]: update the comment at the end of the namespace

File: libs/R-TypeLogic/EntityManipulation/CreateEntitiesFunctions/CreatePlayer.hpp [hunk 16]
```diff
@@ -25,9 +25,12 @@
 #include "R-TypeLogic/Global/Components/VelocityComponent.hpp"
 #include "R-TypeLogic/Global/Components/WeightComponent.hpp"
 #include "R-TypeLogic/Global/SharedResources/Random.hpp"
-#include "R-TypeLogic/Global/Components/LayerLvL.hpp"
 
-namespace ecs
+using namespace rtypelogic::global;
+using namespace transisthor::ecslogic;
+using namespace ecs;
```
Comment (osvegn) [inline libs/R-TypeLogic/EntityManipulation/CreateEntitiesFunctions/CreatePlayer.hpp:31]: Useless

File: libs/R-TypeLogic/EntityManipulation/CreateEntitiesFunctions/CreatePlayer.hpp [hunk 17]
```diff
@@ -25,9 +25,12 @@
...
+
+namespace rtypelogic::entitymanipulation
```
Comment (osvegn) [inline libs/R-TypeLogic/EntityManipulation/CreateEntitiesFunctions/CreatePlayer.hpp:33]: update the comment at the end of the namespace

**Review by osvegn — CHANGES_REQUESTED** [review]
Too much file with the same error, please update all file I had comment and update others that have this error

**Pull request closed without merging**